tests/integration/components/power-select-typeahead-test.js

```diff
@@ -57,3 +60,36 @@ test('Removing a few characters and selecting the same option that is already se
   assert.notOk(find('.ember-power-select-dropdown'), 'The component is closed again');
   assert.equal(find('.ember-power-select-search-input').value, 'Portugal', 'The input contains the selected option');
 });
+
+test('can search async with loading message', function(assert) {
+  this.searchCountriesAsync = () => {
+    return new RSVP.Promise((resolve) => {
+      run.later(() => {
+        resolve(countries);
+      }, 100);
+    });
+  };
+  this.loadingMessage = 'searching...';
+  this.render(hbs`
+    {{#power-select-typeahead 
+      search=searchCountriesAsync
+      selected=selected 
+      loadingMessage=loadingMessage
+      onchange=(action (mut selected)) 
+      extra=(hash labelPath="name") as |country|}}
+      {{country.name}}
+    {{/power-select-typeahead}}
+  `);
+  assert.notOk(find('.ember-power-select-dropdown'), 'The component is closed');
+  typeInSearch('Uni');
+  // TODO: this is failing b/c first load waits to open until options come back
+  // assert.ok(find('.ember-power-select-option--loading-message'), 'The loading message shows');
```

Comment:
Still need to get the loading message to show if `select` is not open and searching.  Only shows when `select` is initially open.